### ISSUE TITLE
QA Ph4: Accessibility_Screen reader does not read the add icons on upcoming modal 

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#922]
With the NVDA screen reader enabled, the "add" icons on the upcoming modal are not read when moused over. 

Steps: 
1. Enable the NVDA screen reader tool 
2. Go to https://io-webapp-staging.appspot.com/io2016/
3. Expand the live modal > go to the upcoming tab 
4. Mouse over the 'add' icons 
5. Note issue 

MS Edge Windows 10

![gio_922](https://cloud.githubusercontent.com/assets/7585604/15256795/0b2556da-18f8-11e6-8a44-275891a21185.PNG)
